metric_tank/usage/usage.go

```diff
@@ -0,0 +1,107 @@
+package usage
+
+import (
+	"fmt"
+	"github.com/benbjohnson/clock"
+	"github.com/raintank/raintank-metric/metric_tank/defcache"
+	"github.com/raintank/raintank-metric/metric_tank/struc"
+	"github.com/raintank/raintank-metric/schema"
+	"sync"
+	"time"
+)
+
+var Clock clock.Clock
+var metrics struc.Metrics
+var defCache *defcache.DefCache
+
+type orgstat struct {
+	keys   map[string]struct{} // track unique keys seen
+	points uint32              // track amount of points seen
+}
+
+// tracks for every org
+type Usage struct {
+	sync.Mutex
+	period time.Duration
+	now    []orgstat
```

Comment:
it doesn't assume that. it does assume that the amount of gaps is relatively low for this approach to be worthwhile, which long term will probably be less and less true, so yeah i'll change for a map.